features/FxA-16-devices-view/README.md

```diff
@@ -4,5 +4,174 @@ Device View - Basic UI
 
 https://mozilla.aha.io/features/FXA-16
 
-As A FF user when using Firefox, I want to know the devices and services I am connected to via my Firefox Account, and important information about my the status of my services. For devices, this includes last time synced, record counts of different data types, and optionally data storage size (perhaps an option for self-hosters or others?)
+Stories
+-------
 
+As a FF user when using Firefox,
+I want to know the devices that are connected to my Firefox Account,
+and important information about the status of those devices.
+
+I want my FF devices to be logically named
+so I recognize them and have the ability to rename them.
+I want to easily see the type and browser details of the each device,
+and to be able to identify the device I am currently using.
+
+I want a place to go when I think that something might be wrong with Sync,
+to see when each device last checked in with the Mozilla servers.
+
+I want a place to go when I think that my account
+may have been used without my permission,
+to check what devices are connected
+and to disconnect devices that I do not recognize.
+
+
+UI Mocks
+--------
+
+@ryanfeeley's mocks go here
+
+
+Details
+-------
+
+#### What is a "device"?
+
+By "device" here we mean "an instance of Firefox".
+There is no separate notion of having multiple "browsers" on a single "device".
+Technically the concept is closer to "profiles"
+but this is not a good name to present the users.
+
+* When I use a desktop machine
+  and connect both Firefox Release and Firefox Nightly to my account,
+  they appear as two separate devices.
+  There will be no indicatiion that they're on the same physical machine
+  apart from similarity in their auto-generated name and their UA details.
+
+* When I use Firefox for Android or Firefox for iOS...
```

Comment:
> instead of building native UI.

Native UI for editing the device name already exists on Desktop and Fennec (although I couldn't see it on iOS).  They just write it into data squirreled away in sync where it's hard for us to get to.